toolkit/utils/db-sync-sqlx/src/lib.rs

```diff
@@ -137,9 +177,12 @@ impl From<sidechain_domain::PolicyId> for PolicyId {
 	}
 }
 
+/// Full identifier of a Cardano native asset
 #[derive(Debug, Clone, sqlx::FromRow, PartialEq)]
 pub struct Asset {
+	/// Minting policy of the asset
```

Comment:
```suggestion
	/// Minting policy id of the asset
```
Policy is before hashing